platform-sdk/swirlds-common/src/main/java/com/swirlds/common/utility/CommonUtils.java

```diff
@@ -377,4 +413,35 @@ public static <T> Consumer<T> combineConsumers(final List<Consumer<T>> consumers
             }
         };
     }
+
+    /**
+     * Returns a string representation of the given byte count in human readable format.
+     *
+     * @param bytes number of bytes
+     * @return human-readable string representation of the given byte count
+     */
+    public static String byteCountToDisplaySize(final long bytes) {
+        if (bytes < 1024) {
+            return bytes + " B";
+        }
+        final int z = (63 - Long.numberOfLeadingZeros(bytes)) / 10;
+        return String.format("%.1f %sB", (double) bytes / (1L << (z * 10)), " KMGTPE".charAt(z));
+    }
+
+    /**
+     * Aligns the given string to the left by padding it with the given character.
+     *
+     * @param str string to pad
+     * @param size size of the resulting string
+     * @param padChar character to pad with
+     *
+     * @return the padded string
+     */
+    public static String leftPad(final String str, final int size, final char padChar) {
+        if (str == null || size <= str.length()) {
+            return str;
+        }
+
+        return String.valueOf(padChar).repeat(size - str.length()) + str;
+    }
```

Comment:
There is a platform utility for this. 

`leftPad("this is my string", 100, ' ')` is equivalent to `ALIGNED_LEFT.pad("this is my string", ' ', 100)`. See `HorizontalAlignmentTests.java` for more examples.